CHANGES.rst

```diff
@@ -12,6 +12,7 @@ Improvements 🙌:
  - Remove jQuery library. Fixes #3381
 
 Other changes:
+ - Fix F-Droid badge and improve formatting and capitalization in README (#3492)
```

Comment:
What is the fix on F-Droid badge?

Reply:
How I think it has vanished after the rebase.

Reply:
Yes, it disappeared with the rebase. The sentence part can be removed from CHANGES.rst.